### PR TITLE
remove user-friendly check for shinx-build in makefile

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- aldryn boilerplate bootstrap3 make install in docs folder

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -1,3 +1,19 @@
+########################################################################################################################
+##### CUSTOM CONFIGURATION
+VENV = env/bin/activate
+PORT = 8000
+# CORE COMMANDS
+install:
+	##### setup virtualenv
```
Comment (ruslanmalogulko) [inline docs/Makefile]: could you pls replace tabs with whitespaces entirely in all file?
and pls check if your .editorconfig properly configured

File: docs/Makefile [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+	virtualenv env
+	. $(VENV); pip install -r requirements.txt
+	. $(VENV); sphinx-build . _build/html
```
Comment (ruslanmalogulko) [inline docs/Makefile]: should you activate virtualenv two times?

File: docs/Makefile (hunk 1 above)
Comment (vxsx) [inline docs/Makefile]: The tab character is a very important part of the syntax of Makefiles. All command lines must start with tabs.
That's why we have this line in `.editorconfig` https://github.com/aldryn/aldryn-boilerplate-bootstrap3/blob/master/.editorconfig#L15

Comment (ruslanmalogulko) [inline docs/Makefile]: Sorry, you're completely right here :)

File: docs/Makefile (hunk 2 above)
Comment (lory87) [inline docs/Makefile]: yes, this is required.

**Comment by FinalAngel** [comment]
LGTM

**Pull request merged**